packages/fxa-content-server/app/scripts/views/settings/delete_account.js

```diff
@@ -33,27 +33,18 @@ var View = FormView.extend({
       });
     }
     this._activeSubscriptions = [];
+    this._uniqueBrowserSessions = [];
     this._hasTwoColumnProductList = false;
     this._productListError = false;
   },
 
-  _formatTitle(items) {
-    return items.map(item => {
-      item.title = item.name;
-      if (item.clientType === CLIENT_TYPE_WEB_SESSION && item.userAgent) {
-        item.title = item.userAgent;
-      }
-      return item;
-    });
-  },
```

Comment:
We no longer need to do this since I'm filtering web session titles in `_setuniqueBrowserSessions()` and I updated the template to just use `name` for `clients`.